AwsCryptographicMaterialProviders/runtimes/java/src/examples/java/software/amazon/cryptography/example/Constants.java

```diff
@@ -5,6 +5,10 @@ public class Constants {
   public static final String BRANCH_KEY_ID = "branch-key-id";
   public static final String TYPE = "type";
   public static final String CREATE_TIME = "create-time";
-  public static final String MUTATION_COMMITMENT = "branch:MUTATION_COMMITMENT";
-  public static final String MUTATION_INDEX = "branch:MUTATION_INDEX";
+  public static final String TYPE_MUTATION_COMMITMENT =
+    "branch:MUTATION_COMMITMENT";
```

Comment:
```suggestion
  public static final String TYPE_MUTATION_COMMITMENT = "branch:MUTATION_COMMITMENT";
```

Reply:
`npx prettier --plugin=prettier-plugin-java --write` will not allow for this.

If we change our Java formater, than we can have this.